src/screens/ReaderScreen.tsx

```diff
@@ -1,24 +1,28 @@
 import * as React from "react";
 import { FunctionComponent } from 'react';
-import { SafeAreaView, StyleSheet } from 'react-native';
+import { SafeAreaView, StyleSheet, useWindowDimensions } from 'react-native';
 import { ReaderScreenProps } from '../navigationTypes';
 import BookReader from "../BookReader";
 
 export const ReaderScreen: FunctionComponent<ReaderScreenProps> = ({ route, navigation }: ReaderScreenProps) => {
+    // Besides height and width, the object also contains scale and fontsize,
+    // should we ever have need of them.
+    const windowSize = useWindowDimensions();
+
+    const styles = StyleSheet.create({
+        safeAreaStyles: {
+            flex: 1,
+            height: windowSize.height,
+            width: windowSize.width,
+            backgroundColor: "#fff",
+        }
+    });
+
     return (
-        <SafeAreaView style={styles.container}>
-            <BookReader bookUrl={route.params.bookUrl} />
+        <SafeAreaView style={styles.safeAreaStyles}>
```

Comment:
Sorry, I regret. container would be a better name for it. (Because if we decided to use a View/ScrollView or whatever, the styles wouldn't really need to change right, so why should we have to rename the style name?)

Reply:
Done.